src/eureka/S6_planet_spectra/s6_spectra.py

```diff
@@ -878,6 +885,180 @@ def compute_amp(meta, log, fit_methods):
     return meta
 
 
+def compute_pc_offset(meta, log, fit_methods):
```

Comment:
This function will currently not work for the `poetpc` model.

Reply:
Correct.  It was going to take more time than I had available this week to also implement it for the POET phase curve model.

Reply:
I think I've come up with a viable implementation based on your implementation for starry.  I'll give it a go and then push another commit.